solr/core/src/java/org/apache/solr/handler/export/FloatValue.java

```diff
@@ -44,6 +44,10 @@ public Object getCurrentValue() {
     return currentValue;
   }
 
+  public void toGlobalValue(SortValue previousValue) {
+
```

Comment:
This PR creates a few empty methods - do these need implemented or are they intentionally empty.

If intentionally empty, may be worth a comment pointing out that they're intentional no-ops?